tasks.rb

```diff
@@ -2338,6 +2338,14 @@ def apply_hpxml_modification_sample_files(hpxml_path, hpxml)
       hpxml_bldg.batteries[0].usable_capacity_kwh = nil
     end
 
+    # ---------------- #
+    # HPXML EV Battery #
+    # ---------------- #
+
+    if ['base-battery-ev-lifetime-model.xml'].include? hpxml_file
```

Comment:
Surprised to see this. We previously disabled this model for home batteries (as part of [this PR](https://github.com/NREL/OpenStudio-HPXML/pull/1223)). I don't remember why offhand without digging in. Are we sure we want to use/allow it here?